examples/prompts/trace_debugging/dev/synthetic_trace_data.ipynb

```diff
@@ -12,18 +43,32 @@
     "from weave.stream_data_interfaces import TraceSpanDict"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "3d0c4b21-9081-4666-ae6f-893230d7b6cf",
+   "metadata": {},
+   "source": [
+    "# Step 1: Configure data streaming"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "c3e49bf5-b40a-4039-85e7-4b4795c0665e",
    "metadata": {},
    "outputs": [],
    "source": [
-    "WB_ENTITY = \"timssweeney\"\n",
-    "WB_PROJECT = \"trace_quickstart\"\n",
-    "WB_STREAM = \"log_stream\"\n",
-    "\n",
-    "st = StreamTable(f\"{WB_ENTITY}/{WB_PROJECT}/{WB_STREAM}\")"
+    "WB_ENTITY = \"\" # replace with your W&B username or team name\n",
```

Comment:
minor/nit: looks like i missed this, set entity to null here as well?

Reply:
done